### ISSUE TITLE
Replace FE_PolyTensor::inverse_node_matrix with something more accurate

### ISSUE THREAD

**Issue by drwells** [dealii/dealii#13837]
If I run the test suite without LAPACK I still get a few failling tests:
```
        3890 - fe/rt_bubbles_12.debug (Failed)
        3891 - fe/rt_bubbles_12.release (Failed)
        3892 - fe/rt_bubbles_13.debug (Failed)
        3893 - fe/rt_bubbles_13.release (Failed)
        3896 - fe/rt_bubbles_15.debug (Failed)
        3897 - fe/rt_bubbles_15.release (Failed)
```
The commonality between these is that all of these tests rely on inverting a relatively large (192 x 192) dense matrix with our own naive Gauss-Jordan algorithm - the result is not so good (I see a maximum difference of 1e-9 between LAPACK and no-LAPACK versions, while the largest value the matrix is about 1).

We should know better than to explicitly invert a matrix this big (and it has a condition number of about 320k) and we should use some other algorithm to compute the inverse without LAPACK (or, better yet, just not use the inverse at all).

**Comment by bangerth** [comment]
Yes, we should know better than to do an explicit inversion. Can you point me at the place where that happens?

**Comment by bangerth** [comment]
That second hunk is in `FE_PolyTensor<dim, spacedim>::shape_value_component()`, which means that we do a matrix-vector product there anytime we ask for shape values. That's kind of expensive, but I guess that happens when you store the wrong basis :-)

What would be a better approach to the conundrum? Build the inverse matrix column by column running GMRES? Requiring LAPACK?
